### DISCUSSION TITLE
OKD 4.15 to 4.16

### DISCUSSION THREAD

**Discussion by dkoci** [okd-project/okd#2017] [Q&A]
Hi.
Will an upgrade from OKD 4.15 to 4.16 be available? The upgrade for OpenShift is already possible, but I notice that the latest version for OKD is 4.15.0-0.okd-2024-03-10-010116.

**Comment by melledouwsma** [comment]
Please see #2012 for the summary on this subject. You can read more details about this on https://okd.io/blog/2024/06/01/okd-future-statement and https://okd.io/blog/2024/07/30/okd-pre-release-testing.

**Reply by dkoci** [reply]
Thank you. I did not know about SCOS and RH saga.

**Comment by madpearl** [comment]
Hi,
 you can use this solution to switch between 4.15 and 4.16. https://github.com/okd-project/okd/discussions/1971#discussioncomment-10119718
 Follow the steps and important to wait kube-apiserver to compete his upgrade. All is in the post explained!

**Reply by dkoci** [reply]
Thank you @madpearl but I will wait for the stable upgrade 🙂.

**Comment by JaimeMagiera** [comment]
Please note that the upgrade from OKD FCOS 4.15 to OKD SCOS 4.16 will require a `--force` upgrade process. It's not going to show up in the list of available updates on the cluster itself.

**Reply by dkoci** [reply]
Will be there any announcement about version which will be suitable for stable upgrade?

**Reply by JaimeMagiera** [reply]
Yes. When we have a GM candidate, with appropriate documentation to match, we'll make an announcement on the blog, in the chat channels, with smoke signals, skywriting, megaphones – anything we can get our hands on.

**Reply by dkoci** [reply]
Great. Thank you for all answers.
